schema/data/fixtures/certifier-access-teardown.sql

```diff
@@ -1,5 +1,7 @@
 begin;
 
+-- Re-populate products table, which was truncated in test setup:
+\i ../schema/data/dev/product.sql
```

Comment:
Same teardown fix as in the [reporter accessibility PR](https://github.com/bcgov/cas-ciip-portal/pull/1013/files#diff-0725859de4bda7644be3d0b5800514b4)